specs/Guid.specs.ts

```diff
@@ -1,4 +1,4 @@
-﻿/// <reference path="Jasmine.d.ts"/>
+/// <reference path="Jasmine.d.ts"/>
```

Comment:
File is deleted

Reply:
Are those references still relevant anyway?